<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScrollArea</class>
 <widget class="QScrollArea" name="ScrollArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scheme</string>
  </property>
  <property name="widgetResizable">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="scrollAreaWidgetContents">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>359</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="addSchemeButton">
        <property name="text">
         <string>Add / Replace Scheme</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="showSchemeButton">
        <property name="text">
         <string>Show Scheme</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteSchemeButton">
        <property name="text">
         <string>Delete Scheme</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filterEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="schemeList"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
